.github/workflows/tests.yml

```diff
@@ -213,7 +212,7 @@ jobs:
           python -m pytest --pyargs lumispy
 
       - name: Run Pyxem Test Suite
-        if: ${{ always() }}
+        # if: ${{ always() }}
+        if: contains(matrix.EXTENSION_VERSION, 'dev') # Tests only against development version, if a fix is not released yet.
```

Comment:
necessary despite `0.17` release that should make pyxem compatible with hs2.0?

Reply:
Yes, but there is an issue with running the test suite with the latest version of pytest. I pinned pytest to .7.4.4 and it works now.